inventory-fusion/src/test/scala/com/normation/inventory/provisioning/fusion/TestReportParsing.scala

```diff
@@ -154,6 +151,11 @@ class TestReportParsing extends Specification with Loggable {
       agents must be empty
     }
 
+    "should have dsc agent agent when using rudder-agent based on dsc" in {
+      val agents = parser.parse("fusion-report/dsc-agent.ocs").node.agents.map(_.name).toList
+      agents == (DSC_AGENT :: Nil)
+    }
+
```

Comment:
Actually there is no field in software that match, we may need to another path in fusion or even in packaging to install rudder-agent to be available in software, i kept the default value "rudder-agent" but it may not be the case